UIInfoSuite2/UIElements/ShowCropAndBarrelTime.cs

```diff
@@ -111,20 +111,17 @@ private void OnUpdateTicked(object? sender, UpdateTickedEventArgs e)
       _currentTerrain.Value = terrain;
     }
 
-    // Make sure that _terrain is null before overwriting it because Tea Saplings are added to terrainFeatures and not IndoorPot.bush
-    if (_currentTerrain.Value != null || _currentTile.Value is not IndoorPot pot)
```

Comment:
The real essence of the change is to remove "_currentTerrain.Value != null ||".  (everything else is just syntax).

As best I can tell, TeaBushes used to work very differently from everything else back in the day.  But now things are a whole lot simpler.  I've been able to test this in several saves with tea bushes but don't have one handy where the tea bushes are supposed to be harvestable.